config/config.json

```diff
@@ -82,6 +82,14 @@
         "ShowEmailAddress": true,
         "ShowFullName": true
     },
+    "LegalAndSupportSettings": {
+        "TermsOfServiceLink": "https://github.com/mattermost/platform/blob/master/README.md",
```

Comment:
The spec [here](https://mattermost.atlassian.net/browse/PLT-881) asked for these to be used as the default (for the time being).  Not against using the static files instead but we might want to have a discussion about the defaults then.